src/features/patients/pages/PatientForm.tsx

```diff
@@ -67,21 +95,26 @@ export const PatientForm: FC = () => {
   const { createPatient, createNameNote } = usePatientApi();
 
   const handleSubmit = () => {
-    if (
-      !(patientFormRef?.current?.isValid && patientFormRef?.current?.isValid)
-    ) {
-      return;
-    }
+    const patientValidator = jsonPatientSchemaValidator();
+    const surveyValidator = jsonPatientSurveySchemaValidator();
+    const patientIsValid = patientValidator(patientFormRef.current?.formData);
+    const surveyIsValid = surveyValidator(surveyFormRef.current?.formData);
```

Comment:
not submitting to often, so just more duplicate validation on the forms.. seemed easiest- the `onChange` which is triggered during a mount of the components doesn't seem to trigger the validation, so the `onChange` doesn't set the form as invalid on the initial mount

Could probably remove the ref `isValid` now.. hm

Reply:
woo! thanks - that's much nicer.
and yes, the `isValid` on the ref can go - which tidies that up a bit too.